gaia-6000/genesis.json

```diff
@@ -72,101 +72,202 @@
+                },
+                "power":1000,
+                "name":"kwunyeung"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "vSr94aI+zfF3D8Cr2VtCXPpfgj7t2ck8SlZxRsfn7gk="
+                },
+                "power":1000,
+                "name":"figmatt"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "p5ijoVwp2zmA7RkXXvPl+yqdnlaWMwoV2pYIN8bDyFs="
+                },
+                "power":1000,
+                "name":"jla-bsd"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "KOdWpo4aQFrLxVlkyc66p7m6mBNnPLmGuO4Z4L+CI1Y="
+                },
+                "power":1000,
+                "name":"Gold"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "VfOsmcON77auerRc9zKwOR+CvL0sj1nA45hS2WqX1xE="
+                },
+                "power":1000,
+                "name":"nylira" 
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "QYONAknaJqx3OKPSKraDrOPkx6xssezYtXVS84nZvZE="
+                },
+                "power":1000,
+                "name":"BKCM"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "PyFJmNrUres3QOkj2BjxplCxrOF+HDFGohi3tRKsToY="
+                },
+                "power":1000,
+                "name":"Dev's Validator"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "s4ER09+WeX10euzGyK7xDW7+myQVXt3Plup8IHUE4nk="
+                },
+                "power":1000,
+                "name":"Staking Facilities"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "OPxj9edXgufNEjYNhZKqLgmYnK4A3nGw3rxaFQrHn24="
+                },
+                "power":1000,
+                "name":"nuevax"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "Hi2rtbdJdQtOe3Kq4OoD/xkWJbIjIsUI9qgLQ6TlhiM="
+                },
+                "power":1000,
+                "name":"vultr.guest"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "u/Ra5oGuTpWJycIjuI+jAc4hqJOU5yBV29RzdyL0YAc="
+                },
+                "power":1000,
+                "name":"gnitset"
+            },
+            {
+                "pub_key": {
+                    "type": "AC26791624DE60",
+                    "value": "ah3JtmBA7gbxSiimPsLqQlV85gpNOUBJMvnxGx8eVlo="
```

Comment:
Is it an error that my node pub_key appear twice here? Was it because I declare two validator candidates using the same pub_key before? https://github.com/cosmos/cosmos-sdk/issues/1010

Reply:
I am gonna delete the duplicate key